File: play-book-api/organization-service/src/main/java/org/atlas/requests/SpaceUpdateRequest.java
package org.atlas.requests;

import lombok.Data;
import org.atlas.enums.SpaceVisibility;
import org.springframework.data.relational.core.mapping.Column;

import java.util.UUID;
@Data
public class SpaceUpdateRequest {
    private UUID space_id;
    private String name;
    private String description;
    private String icon;
    private UUID teamId;
    private SpaceVisibility visibility;
}

File: play-book-ui/src/app/core/requests/space-update.request.ts
import {SpaceVisibilityEnum} from '../../store/models/enums/space-visibility.enum';

import {SpaceMembersRequest} from './space-members.request';
import {SpaceMember} from '../../store/models/space-members.model';

export interface SpaceUpdateRequest {
  space_id: string;
  name: string;
  teamId: string;
  description: string;
  icon: string;
  members: SpaceMember[]
  visibility: SpaceVisibilityEnum


}
